File: cpp/764_calculate_circumference_and_area.cpp
class Solution {
public:
    /**
     * @param r: a Integer represent radius
     * @return: the circle's circumference nums[0] and area nums[1]
     */
    vector<double> calculate(int r) {
        const double PI = 3.14;
        vector<double> v;
        v.push_back(2 * PI * r);
        v.push_back(PI * r * r);
        return v;
    }
};

File: Java/764_calculate_circumference_and_area.java
public class Solution {
    /**
     * @param r: a Integer represent radius
     * @return: the circle's circumference nums[0] and area nums[1]
     */
    public double[] calculate(int r) {
        final double PI = 3.14;
        double[] result = new double[2];
        result[0] = 2 * PI * r;
        result[1] = PI * r * r;
        return result;
    }
}
